src/modelfree/multi/score.py

```diff
@@ -91,6 +95,85 @@ def multi_score(score, save_path):
     return {'scores': results, 'exp_id': exp_id}
 
 
+def run_external(named_configs, post_named_configs, config_updates, adversary_path=None):
+    """Run multiple multi_score experiments. Intended for use by external scripts,
+       not accessible from commandline.
+
+       :param named_configs: (list<str>) list of named configs, executed one by one
+       :param post_named_configs: (list<str>) list of named configs, always applied.
```

Comment:
The combination of (docstring + variable names) doesn't really give me a clear idea of what's happening here (which my understanding from looking at the code is "some set of baseline configs that is added to each specific experiment config")

Reply:
I've tried to clarify this in the docstring.